pages/Documents/Reporting/OutboundConfigurationAPI/overview.md

```diff
@@ -0,0 +1,400 @@
+       {
+           "type": "body",
+           "content": "Your ticket for ${{1}}\n\nTime - ${{2}}\nVenue - ${{3}}\nSeats - ${{4}} - ${{5}}",
+           "variables": [
+               "1",
+               "2",
+               "3",
+               "4",
+               "5"
+           ]
+       }
+  ]
+}
+```
+  Whats app rich content response example
+```json
+{
+   "type": "wa",
+   "language": "en",
+   "namespace": "b8520e15_f77b_5f76_fb50_9f756d03676b",
+   "templatename": "newtemplate_button_cta_feb14",
+   "outboundnumber": "12535277322",
+   "messageComponents": [
+                             {
+                                 "type": "body",
+                                 "content": "Your ticket for ${{1}}\n\nTime - ${{2}}\nVenue - ${{3}}\nSeats - ${{4}} - ${{5}}",
+                                 "variables": [
+                                     "1",
+                                     "2",
+                                     "3",
+                                     "4",
+                                     "5"
+                                 ]
+                             },
+                             {
+                                 "type": "footer",
+                                 "content": "Just show this QR code at the cinema"
+                             },
+                        ]
+}
+```
+
+The messageComponents fields can reflect the following fields depending on the template
+
+| Header image with static url | {<br /> &nbsp;  "type": "header",<br /> &nbsp;  "sub_type": "image",<br />&nbsp;   "content": "https://upload.wikimedia.org/commons-1/c/ce/1963_Tornadoes.png",<br /> &nbsp;  "variables": []<br />}|
+| Header image with dynamic url that is expected to be passed in the request payload | {<br /> &nbsp;  "type": "header",<br /> &nbsp;  "sub_type": "image",<br /> &nbsp;   "content": "${{1}}",<br /> &nbsp;  "variables": [ "1" ]<br />} |
+| Header with a static document url | {<br /> &nbsp;  "type": "header",<br /> &nbsp;  "sub_type": "document",<br /> &nbsp;  "content": "https://cdn.timelab.se/cellite-1/20180626140206/{{1}}.pdf",<br /> &nbsp;  "variables": [ "1" ]<br />} |
+| Header with a static video url | {<br /> &nbsp;  "type": "header",<br /> &nbsp;  "sub_type": "video",<br /> &nbsp;  "content": "https://file-examples-com.github.io/uploads/2017/04/file_example_MP4_640_3MG.mp4",<br /> &nbsp;  "variables": []<br /> } |
+| Button with a static call to action  value | {<br /> &nbsp;  "type": "button",<br /> &nbsp;  "index": 0,<br /> &nbsp;  "content": "Call",<br /> &nbsp;  "sub_type": "phone_number",<br /> &nbsp;  "phone_number": "+12062061111"<br /> } |
+| Button with a dynamic call to action value  that is expected to be passed in the request payload | {<br /> &nbsp;  "type": "button",<br /> &nbsp;  "index": 1,<br /> &nbsp;  "content": "Link",<br /> &nbsp;  "sub_type": "url",<br /> &nbsp;  "url": "https://www.google.com/{{1}}",<br /> &nbsp;  "variables": [ "1" ]<br /> } |
```

Comment:
Try to minimize the use of br and HTML in markdown.

If you use it, you can use it without / only <br>

Reply:
Done